sdk/firstload/package.json

```diff
@@ -12,6 +35,7 @@
 	},
 	"type": "module",
 	"main": "./dist/index.js",
+	"browser": "./dist/index.umd.cjs",
```

Comment:
@Vadman97 unfortunately this will also result in most esm-supporting dev tools and bundlers to use the umd build instead of the esm build, which is probably not what we want.

I think a better approach would be to add individual `unpkg` and `jsdelivr` keys for the umd build instead. 

Or possibly drop the umd build altogether and ask users to start importing these as modules from the cdns.

Reply:
Even with the exports field I think the existence of the browser field will force bundlers to switch to the umd entry point, I believe. That's the intended purpose of the browser field after all.

I think named exports getting broken is very likely a result of the umd module being used instead of esm as well. So removing the browser field should hopefully fix both.

Reply:
Here's the defacto spec for the browser field that most dev tools adhere to, for reference: https://github.com/defunctzombie/package-browser-field-spec